src/Runner.Common/Constants.cs

```diff
@@ -219,14 +219,15 @@ public static class Actions
                 public static readonly string AllowUnsupportedStopCommandTokens = "ACTIONS_ALLOW_UNSECURE_STOPCOMMAND_TOKENS";
                 public static readonly string RunnerDebug = "ACTIONS_RUNNER_DEBUG";
                 public static readonly string StepDebug = "ACTIONS_STEP_DEBUG";
+                public static readonly string AllowActionsUseUnsecureNodeVersion = "ALLOW_ACTIONS_USE_UNSECURE_NODE_VERSION";
```

Comment:
nit can we do `ACTIONS_ALLOW...` as all of our other env's start with `ACTIONS_`

Reply:
`ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION`